### DISCUSSION TITLE
HA-addon : log  directroy meter_reading

### DISCUSSION THREAD

**Discussion by chpego** [wmbusmeters/wmbusmeters#760] [General]
Hi guys !

I'm curious, but i'm looking for logs of the daemon wmbusmeters:
But there is no log, because in this line, you're using useconfig options
https://github.com/weetmuts/wmbusmeters/blob/f992c9f8107d2766c72f9dcb3aa51f6360b50131/ha-addon/run.sh#L60
and accordling with [this](https://github.com/weetmuts/wmbusmeters#run-using-config-files) , we cannot use this few options:
> The option `--useconfig=` can only be combined with a few other options: `--overridedevice= --listento= --exitafter= --oneshot= --silent --normal --verbose --debug --trace`

So the directory logs/meter_readings is empty, maybe adding this in **run.sh** can help to have logs of meter:
``
echo -e "meterfiles=$CONFIG_DATA_PATH/logs/meter_readings" >> $CONFIG_DATA_PATH/etc/wmbusmeters.conf
``

What do you think?

**Comment by BIBOLV** [comment]
Hi,
Now with new config interface you can change config per your wishes, but what would be use case for having meter readings in file, because all logs are being sent to HA and they are available from HA side, but readings are meant to be sent to mqtt to display in HA.
